modules/targets/darwin/default.nix

```diff
@@ -48,6 +50,22 @@ in {
         platforms.darwin)
     ];
 
+    warnings = let
+      batteryPercentage =
+        attrByPath [ "com.apple.menuextra.battery" "ShowPercent" ] null
+        cfg.defaults;
+      webkitDevExtras = attrByPath [
+        "com.apple.Safari"
+        "com.apple.Safari.ContentPageGroupIdentifier.WebKit2DeveloperExtrasEnabled"
+      ] null cfg.defaults;
+    in optional (batteryPercentage != null) ''
+      This option no longer works on macOS 11 and later. Instead, open System
+      Preferences, go to "Dock &amp; Menu Bar", select "Battery", and toggle
+      the checkbox labeled "Show Percentage."
+    '' ++ optional (webkitDevExtras != null) ''
+      This option is no longer present in recent versions of Safari.
+    '';
```

Comment:
Maybe the option name should also be mentioned here?

Reply:
Oops, I've now included the option name.